src/test/resources/com/znsio/sample/e2e/features/amazon_cart.feature

```diff
@@ -0,0 +1,10 @@
+@amazon
+Feature: Amazon
+
+  @web @amazon @amazon-test
+  Scenario: Add to cart
+    Given I launch amazon application
+    And I search for "Iphone 13" and verify results
+    When I Open first search result details
```

Comment:
This should have been `When I open first search result details`

`When I Open first search result details` -> what does "open first search result details" mean?

Reply:
ex: `When I select the first product from the search results`

Reply:
Done